### PR TITLE
[TOOLS-2038] Highcharts charts adjustments

### PR DESCRIPTION
**Pull request by marcokuchla**
* Ticks using min pixel size to avoid rotation;
 * Removing title from charts by default;
 * Removing chart's title as key when registering chart in dc-date-range-service;

### CODE REVIEW

File: src/main/resources/static/scripts/dc-date-range-charts-service.js [hunk 1]
```diff
@@ -106,12 +106,12 @@ function DcDateRangeChartsService() {
...
-            self.applySelection(chart.title.textStr)
-        })
+            self.applySelection(chart);
+        });
     };
 
-    self.applySelection = function(chartName) {
-        const chart = self._highchartCharts.get(chartName);
+    self.applySelection = function (chart) {
+        // const chart = self._highchartCharts.get(chartName);
```
Comment (objective-solutions-user) [inline src/main/resources/static/scripts/dc-date-range-charts-service.js]: MAJOR Remove this commented out code. [rule](http://10.44.1.250:9000/coding_rules#rule_key=javascript%3ACommentedCode)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/main/resources/static/elements/taskboard/kpis/widget-wip.html [hunk 2]
```diff
@@ -210,15 +210,15 @@ <h3 class="tb-label">Level</h3>
                 _createChart: function () {
                     const series = this._transformToHighchartsSeries(this._plotData.statusesGroup, this._plotData.issueStatusesOrdered);
                     const builder = new WeeklyChartBuilder('wip')
-                        .withTitle('WIP')
                         .withChartType('column')
                         .withTooltipNumberOfDecimals(2)
+                        .withTooltipHeaderFormat('<span style="font-size: 10px">{point.key}</span><br/><span style="font-size: 0.9em">Values are shown in average</span><br/>')
```
Comment (jhonypg) [inline src/main/resources/static/elements/taskboard/kpis/widget-wip.html]: Could you transform this inline style in a class and remove the BR tag?

Comment (marcokuchla) [reply src/main/resources/static/elements/taskboard/kpis/widget-wip.html]: Suggestion accepted.

**Pull request merged**